mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule CachedContentful.Mixfile do
   def project do
     [
       app: :cached_contentful,
-      version: "0.4.1",
+      version: "0.5.0",
```

Comment:
Maybe next time make it version 1, the product is already in use in a production deploymebnt

Reply:
Good point!